scripts/spack/devtools_configs/toss_3_x86_64_ib/packages.yaml

```diff
@@ -89,15 +115,15 @@ packages:
     externals:
     - spec: pkg-config
       prefix: /usr
-  tar:
+  readline:
```

Comment:
This looks like there's a story behind it...

Reply:
spack built like 3x the amount of packages as before...

Reply:
Just wait till you see all the changes that were needed to the `packages.yaml` to work with Spack's new concretizer...